.github/workflows/phpstan.yml

```diff
@@ -0,0 +1,47 @@
+name: PHPStan
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    name: PHP ${{ matrix.php-versions }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.3', '7.4']
```

Comment:
Do we really need to run PHPStan with two different PHP versions or would one be sufficient?

Reply:
I think one phpversion is sufficient since phpstan is not using php to analyze but uses it's own metadata to know about native functions.